### PR TITLE
added tiggers for E

### PR DESCRIPTION
**Pull request by fatima99s**

### CODE REVIEW

File: plugins/aws/ec2/allowedCustomPorts.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ module.exports = {
             default: ''
         }
     },
+    realtime_triggers: ['ec2:AuthorizeSecurityGroupIngress', 'ec2:ModifySecurityGroupRules'],
```
Comment (muzzamilinovaqo) [inline plugins/aws/ec2/allowedCustomPorts.js:21]:
```suggestion
    realtime_triggers: ['ec2:CreateSecurityGroup,ec2:AuthorizeSecurityGroupIngress', 'ec2:ModifySecurityGroupRules'],
```

File: plugins/aws/emr/emrClusterLogging.js [hunk 2]
```diff
@@ -10,6 +10,7 @@ module.exports = {
     link: 'https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-plan-debugging.html',
     recommended_action: 'Modify EMR clusters to enable cluster logging',
     apis: ['EMR:listClusters', 'EMR:describeCluster'],
+    realtime_triggers: ['emr:CreateCluster'],
```
Comment (muzzamilinovaqo) [inline plugins/aws/emr/emrClusterLogging.js:13]:
```suggestion
    realtime_triggers: ['emr:CreateCluster','emr:TerminateJobFlows'],
```

File: plugins/aws/emr/emrEncryptionAtRest.js [hunk 3]
```diff
@@ -10,6 +10,7 @@ module.exports = {
     link: 'https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-data-encryption-options.html',
     recommended_action: 'Update security configuration associated with EMR cluster to enable encryption at rest for local disks.',
     apis: ['EMR:listClusters', 'EMR:describeCluster', 'EMR:describeSecurityConfiguration'],
+    realtime_triggers: ['emr:CreateCluster', 'emr:CreateSecurityConfiguration'],
```
Comment (muzzamilinovaqo) [inline plugins/aws/emr/emrEncryptionAtRest.js:13]:
```suggestion
    realtime_triggers: ['emr:CreateCluster', 'emr:CreateSecurityConfiguration','emr:DeleteSecurityConfiguration'],
```

File: plugins/aws/emr/emrEncryptionInTransit.js [hunk 4]
```diff
@@ -10,6 +10,7 @@ module.exports = {
     link: 'https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-data-encryption-options.html',
     recommended_action: 'Update security configuration associated with EMR cluster to enable encryption in transit.',
     apis: ['EMR:listClusters', 'EMR:describeCluster', 'EMR:describeSecurityConfiguration'],
+    realtime_triggers: ['emr:CreateCluster', 'emr:CreateSecurityConfiguration'],
```
Comment (muzzamilinovaqo) [inline plugins/aws/emr/emrEncryptionInTransit.js:13]:
```suggestion
    realtime_triggers: ['emr:CreateCluster', 'emr:CreateSecurityConfiguration','emr: DeleteSecurityConfiguration'],
```

File: plugins/aws/eventbridge/eventBusCrossAccountAccess.js [hunk 5]
```diff
@@ -37,6 +37,7 @@ module.exports = {
             default: 'aws:PrincipalArn,aws:PrincipalAccount,aws:PrincipalOrgID,aws:SourceAccount,aws:SourceArn,aws:SourceOwner'
         },
     },
+    realtime_triggers: ['eventbridge:CreateEventBus'],
```
Comment (muzzamilinovaqo) [inline plugins/aws/eventbridge/eventBusCrossAccountAccess.js:40]:
```suggestion
    realtime_triggers: ['eventbridge:CreateEventBus','eventbridge:PutPermission'],
```

File: plugins/aws/eventbridge/eventBusPublicAccess.js [hunk 6]
```diff
@@ -19,6 +19,7 @@ module.exports = {
             default: 'aws:PrincipalArn,aws:PrincipalAccount,aws:PrincipalOrgID,aws:SourceOwner,aws:SourceArn,aws:SourceAccount'
         }
     },
+    realtime_triggers: ['eventbridge:CreateEventBus', 'eventbridge:PutRule', 'eventbridge:PutTarget'],
```
Comment (muzzamilinovaqo) [inline plugins/aws/eventbridge/eventBusPublicAccess.js:22]:
```suggestion
    realtime_triggers: ['eventbridge:CreateEventBus','eventbridge:PutPermission', 'eventbridge:PutRule', 'eventbridge:PutTarget'],
```

File: plugins/aws/eventbridge/eventsInUse.js [hunk 7]
```diff
@@ -10,6 +10,7 @@ module.exports = {
     link: 'https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-rules.html',
     recommended_action: 'Create EventBridge event rules to meet regulatory and compliance requirement within your organization.',
     apis: ['EventBridge:listRules'],
+    realtime_triggers: ['eventbridge:PutRules', 'eventbridge:EnableRule', 'eventbridge:DeleteRule'],
```
Comment (mehakseedat63) [inline plugins/aws/eventbridge/eventsInUse.js:13]:
```suggestion
    realtime_triggers: ['eventbridge:PutRule', 'eventbridge:EnableRule', 'eventbridge:DeleteRule'],
```

File: plugins/aws/elb/elbHttpsOnly.js [hunk 8]
```diff
@@ -20,7 +20,7 @@ module.exports = {
     apis_remediate: ['ELB:describeLoadBalancers'],
     actions: {remediate: ['ELB:deleteLoadBalancerListeners'], rollback: ['ELB:createLoadBalancerListeners']},
     permissions: {remediate: ['elasticloadbalancing:DeleteLoadBalancerListeners'], rollback: ['elasticloadbalancing:CreateLoadBalancerListeners']},
-    realtime_triggers: ['elasticloadbalancing:CreateLoadBalancerListeners','elasticloadbalancing:CreateLoadBalancer'],
+    realtime_triggers: ['elb:CreateLoadBalancerListeners','elb:CreateLoadBalancer', 'elb:DeleteLoadBalancer', 'elb:DeleteLoadBalancerListeners'],
```
Comment (mehakseedat63) [inline plugins/aws/elb/elbHttpsOnly.js:23]:
```suggestion
    realtime_triggers: ['elb:CreateLoadBalancerListeners','elb:CreateLoadBalancer', 'elb:DeleteLoadBalancer', 'elb:DeleteLoadBalancerListeners'],
```
```suggestion
    realtime_triggers: ['elb:CreateLoadBalancerListeners','elb:CreateLoadBalancer', 'elb:DeleteLoadBalancer', 'elb:DeleteLoadBalancerListeners'],
```
why did we change it from 'elasticloadbalancing' to 'elb'?

**Review by mehakseedat63 — CHANGES_REQUESTED** [review]
need to verify elasticloadbalancing events

**Review by mehakseedat63 — APPROVED** [review]

**Pull request merged**